tests/conftest.py

```diff
@@ -105,10 +106,16 @@ def pip_lock_file() -> None:
     yield
 
 
-@pytest.fixture(scope="function", params=[7, 8])
+@pytest.fixture(
+    scope="function",
+    params=glob.glob(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))) + "/dockerfiles/*"
```

Comment:
```suggestion
        os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "dockerfiles", "*")
```
Best to always use `os.path.join` to build paths. Among other things it makes sure to take differences between OS' into account.